.gitmodules

```diff
@@ -5,4 +5,4 @@
 [submodule "extension-ci-tools"]
 	path = extension-ci-tools
 	url = https://github.com/duckdb/extension-ci-tools
-	branch = main
```

Comment:
I keep seeing changes surrounding this submodules, do we need to keep modifying them?

Reply:
And this change looks strange. I'll make sure that I'm not modifying this file in the future commits.